src/providers/execution/contracts/CSModule.py

```diff
@@ -1,72 +1,92 @@
 import json
 import logging
 from itertools import groupby
-from typing import Callable, Iterable, cast
+from typing import Callable, Iterable, NamedTuple, cast
 
 from eth_typing import BlockNumber, ChecksumAddress
 from web3.contract.contract import Contract, ContractEvent
 from web3.types import BlockIdentifier, EventData
 
 from src import variables
+from src.web3py.extensions.lido_validators import NodeOperatorId
 
 logger = logging.getLogger(__name__)
 
 
+# TODO: Move to types?
+class NodeOperatorSummary(NamedTuple):
+    targetLimitMode: int
+    targetValidatorsCount: int
+    stuckValidatorsCount: int
+    refundedValidatorsCount: int
+    stuckPenaltyEndTimestamp: int
+    totalExitedValidators: int
+    totalDepositedValidators: int
+    depositableValidatorsCount: int
+
+
 class CSModule(Contract):
     abi_path = "./assets/CSModule.json"
 
-    MAX_OPERATORS_COUNT = 2 ** 64
+    MAX_OPERATORS_COUNT = 2**64
 
     # TODO: Inherit from the base class.
     def __init__(self, address: ChecksumAddress | None = None) -> None:
         with open(self.abi_path, encoding="utf-8") as f:
             self.abi = json.load(f)
         super().__init__(address)
 
-    def get_stuck_node_operators(self, l_block: BlockIdentifier, r_block: BlockIdentifier) -> Iterable:
+    # TODO: Consider adding cache.
+    def current_stuck_operators_ids(self, block: BlockIdentifier = "latest") -> Iterable[NodeOperatorId]:
```

Comment:
`current...` sounds a little misleading. can it be just `get_stuck_operators_ids` ?